test/NumberContainer.java

```diff
@@ -0,0 +1,23 @@
+package test;
+
+import java.util.List;
+
+public class NumberContainer {
```

Comment:
>Create class-container for numbers;

Try to do it for single number (not collection), but with possibility to use any number (Double, Float, etc) not only Integer.